golang/testsuite/services_impl/api/api_container_config_factory.go

```diff
@@ -4,14 +4,13 @@ import (
 	"encoding/json"
 	"fmt"
 	"github.com/kurtosis-tech/kurtosis-client/golang/services"
-	"github.com/kurtosis-tech/kurtosis-libs/golang/testsuite/services_impl/datastore"
 	"github.com/palantir/stacktrace"
 	"github.com/sirupsen/logrus"
 	"os"
 )
 
 const (
-	port = 2434
+	Port = 2434
```

Comment:
Not suuuper happy about this being public and breaking the abstraction, but then again we'll be tearing apart the `ContainerConfigFactory` soon so no need to change it

Reply:
ok